utils/withTheme.js

```diff
@@ -0,0 +1,10 @@
+import { ThemeProvider } from 'styled-components'
```

Comment:
Instead of having to include this file in every component and wrap each manually, we can simply wrap `_app.js`. This will make theme variable available to the whole application.

Reply:
This is needed only to mock Storybook and Jest renders since they don't use `_app.js`

Reply:
Got it, makes sense. However I'd like it if we can avoid to wrap each storybook component with a provider, what do you think about this: http://ryanlanciaux.com/blog/2019/08/20/use-provider-in-storybook/